app/dashboard/models.py

```diff
@@ -3570,9 +3569,9 @@ def name(self):
         if self.admin_override_name:
             return self.admin_override_name
 
-        if self.data and self.data["name"]:
+        # TODO: investigate how jsonfield blank keys get set.
+        if self.data and self.data["name"] and self.data["name"] != 'null':
```

Comment:
should we use `.get` here or inspect the keys directly? i.e. `"name" in self.data.keys()

Otherwise if name doesn't exist it will throw an exception

Reply:
I'm not entirely sure what i am attempting here, this was created during the hackathon sponsor profiles stuff since i branched off of that work for the townsquare stuff, lets discuss.

Reply:
going through it, turns out there were cases that the value set to data was a string saying null